### DISCUSSION TITLE
Task - Spring Boot - Spring Security - Nuclear Fallout

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#76] [Tasks]
Create a Spring Boot application named `nuclear-missiles-app`.

Create a resource named `NuclearResource` that has a handler method `launchMissiles()` mapped to the route `/lauch/missiles`.

Using Spring Security, configure the application such that only the user with the name "president" and password "armageddon" has access to the application.

Run the application and try it out.

### To be submitted:
- Controllers
- application.properties
- Screenshots

**Comment by Svishva** [comment]
## NuclearResource .java
```java
@Controller
public class NuclearResource {

	@RequestMapping("/lauch/missiles")
	public @ResponseBody String launchMissiles() {
		return "Missile Launched";
	}
}

```

## application.properties
```properties
spring.security.user.name=president
spring.security.user.password=armageddon
```

![image](https://user-images.githubusercontent.com/13867858/168603360-685d46a0-0fd4-44c3-bb35-be4fb5b6d9de.png)

![image](https://user-images.githubusercontent.com/13867858/168603405-8a27400b-8091-4ceb-8947-28e6993683ed.png)
